unified-codes/libs/data/src/lib/v2/entities/FormEntity.ts

```diff
@@ -1,7 +1,7 @@
 import { EEntityType, IEntity, Entity } from './Entity';
 import { IFormQualifierEntity, FormQualifierEntity } from './FormQualifierEntity';
 import { IUnitEntity, UnitEntity } from './UnitEntity';
-import { IProperty, Property } from '../Property';
+import { IProperty, Property } from '../properties';
```

Comment:
Nit: since these imports both coming from `'../properties/Property'`, may as well directly import from there? E.g.

```
import { IProperty, Property } from '../properties/Property';
```

Reply:
Hmm yeah, why not - done with b0cb9ddd2.
Let's see how the next CI run goes 🤞